package.json

```diff
@@ -5,7 +5,8 @@
   "main": "lib/fastifyCsrf.js",
   "types": "lib/fastifyCsrf.d.ts",
   "scripts": {
-    "test": "tap test/*.js --cov"
+    "test": "tap test/*.js --cov && npm run typescript",
+    "typescript": "tsd"
```

Comment:
Unfortunately, the naming here seems a little misleading and confusing. The script itself seems superfluous since it does not configure `tsd` using env variables or arguments. Could you change the `test` script to `tap test/*.js --cov && tsd`, please?

Reply:
Yep you're right. I changed it to your suggestion. See 19cea1d